docs/core/install/includes/linux-ubuntu-apt-prep-intro.md

```diff
@@ -0,0 +1,11 @@
+
+Installing with APT (Advanced Package Tool) can be done with a few commands. Before installing .NET, you'll need to:
+
+- Add the Microsoft package signing key to the list of trusted keys.
+- Add the repository to the package manager.
+- Install required dependencies.
+
+> [!TIP]
+> Registering the key only needs to be done once per machine.
+
+Open a terminal and run the following commands:
```

Comment:
This can be misinterpreted as sending you off to do those three things before proceeding. (We've got direct feedback from one person who saw it this way, but I don't think we did anything at that time to address it.) I think adding the tip will make it worse: it visually breaks it up even more, and since it's pointing at one of the three above steps and telling you something about it, it implies adding the key is something you have to actually do. But in Ubuntu, the package repo setup package does that for you.

Is the preamble necessary? Can we be more specific that we're describing what the command does, if we do need to describe it?

(The tip also adds the touchy word "register" back in that we removed earlier.)

Reply:
OK, I reworded this section, can you review it again?